src/blocks/skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,43 @@
+import { type FC } from 'react';
+import styled from 'styled-components';
+
+import type { SkeletonProps } from './Skeleton.types';
+import { getSkeletonPulseAnimation, getSkeletonResponsiveCSS } from './Skeleton.utils';
+import { skeletonCSSPropsKeys } from './Skeleton.constants';
+
+const StyledSkeleton = styled.div.withConfig({
+  shouldForwardProp: (prop, defaultValidatorFn) =>
+    !skeletonCSSPropsKeys.includes(prop as keyof SkeletonProps) && defaultValidatorFn(prop),
+})<SkeletonProps>`
+  /* Responsive props */
+  ${(props) => getSkeletonResponsiveCSS(props)}
+
+  /* Extra CSS prop */
+  ${(props) => props.css || ''}
+
+  /* Animation props */
+  animation: ${getSkeletonPulseAnimation('hsl(200, 20%, 80%)', 'hsl(200, 20%, 95%)')} 1s linear infinite alternate;
+
+  /* Hide children */
+  & > * {
+    visibility: hidden !important;
+  }
+`;
+
+const Skeleton: FC<SkeletonProps> = ({ isLoading, children, ...rest }) => {
```

Comment:
```suggestion
const Skeleton: FC<SkeletonProps> = ({ borderRadius="r2", children,  isLoading, ...rest }) => {
```

Let's at least use a default value of  borderRadius (which is used often in the codebase) to avoid passing it every time we use the component.

And also de structure  props in alphabetical order